examples/UEFA/UEFA.en.shtml

```diff
@@ -113,6 +113,8 @@ class Token {
             redeemedMessage = "(Redeemed)"
         } else if(this.props.redeemed === "0") {
             redeemedMessage = "(Not redeemed)"
+        } else {
+            redeemedMessage = "Redeem Error"
```

Comment:
@developerpeachy can we have it that it simply displays nothing if not redeemed or there is an error?